src/app/public/modules/timepicker/timepicker.interface.ts

```diff
@@ -1,9 +1,37 @@
 export interface SkyTimepickerTimeOutput {
+
+  /**
+   * Specifies the hour.
+   */
   hour: number;
+
+  /**
+   * Specifies the minute.
+   */
   minute: number;
+
+  /**
+   * Specifies the meridian.
```

Comment:
I don't know what this is and couldn't find it on a quick Google search. But I did find that there is another term I'm unfamiliar with: `meridiem`. That term looks closer to the property name of `meridie`, so 2 questions:
1. Are we sure this should be `meridian` and not `meridiem`?
2. Should we define whichever term it the correct one to include here? Or does it not really matter

Reply:
I'd lean on you for that answer... I was not aware of this term either before looking at this code. What do you think our audience would respond to the most?

Reply:
I have no idea on this one (it's not in the existing docs), and I don't know how much it really matters. I mean, it strikes me as a fringe configuration option that we might have included just because it was included in something else that we were using as a model. I don't necessarily see anyone being confused by the property because I don't necessarily see anyone using it, but that's just a guess. And if a consumer wants to understand this property, our property description here (and even the property name) doesn't really help. https://github.com/blackbaud/skyux-datetime/blob/master/src/app/public/modules/timepicker/timepicker.interface.ts doesn't give any insight into how this property is used, and I don't know where to go from there in the code.

I checked the history for the interface at https://github.com/blackbaud/skyux-datetime/commits/master/src/app/public/modules/timepicker/timepicker.interface.ts, and it doesn't look like anyone has touched it since Conor and Steve ported it over to this repo. 

What do you think? I don't want to get hung up and delay the entire PR for one property description that may not really matter, but I don't think the description we have is clear or useful, so I'd like to fix it at some point.

Reply:
I updated the property description as per our discussion on Slack, so resolving this conversation...